lambdas/ssm-parameters/src/ssm-parameters-handler.ts

```diff
@@ -0,0 +1,38 @@
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { getParametersByName } from "@aws-lambda-powertools/parameters/ssm";
+import { Parameter } from "@aws-sdk/client-ssm";
+
+const PARAMETER_TTL =
+  parseNumber(process.env.POWERTOOLS_PARAMETERS_MAX_AGE) ?? 300;
+
+function parseNumber(value?: string) {
+  return parseInt(value ?? "", 10) || undefined;
+}
```

Comment:
`PARAMETER_TTL` is not strictly static, so it should be `parameter_ttl`. I know I have done this in the past. Also I don't thing you need a function

`const PARAMETER_TTL =
  Number(process.env.POWERTOOLS_PARAMETERS_MAX_AGE) || 300;` should do the job